software/control_sw/src/blocks/fpga.py

```diff
@@ -157,6 +173,7 @@ def get_status(self):
         stats['programmed'] = self.is_programmed()
         stats['timestamp'] = datetime.datetime.now().isoformat()
         stats['host'] = '%s:%s' % (self.server_hostname, self.host.host)
+        stats['antname'] = self.antname
```

Comment:
Unless the antname attribute is defaulted somewhere, isn't this command going to fail if it is called before `set_connected_antname`. It should probably return None or NOTSET, or somesuch